Csn.Public.Dto/Lead/Lead.cs

```diff
@@ -27,33 +27,22 @@ public Lead()
         public string Status { get; set; }  
         
         public Environment Environment { get; set; }
-        public Prospect Prospect { get; set; }
+        public Customer Prospect { get; set; }
 
         public string Comments { get; set; }
         public DateTime CreatedUtc { get; set; }
         public DateTime? ModifiedUtc { get; set; }
 
-        public List<TypeValue> Tags { get; set; }
+        public List<TypeValue> Labels { get; set; }
         public CallConnect CallConnect { get; set; }
 
-        public LeadVehicle Item { get; set; }
-        public LeadVehicle TradeIn { get; set; }
+        public List<Vehicle.Vehicle> Items { get; set; }

```

Comment:
Perhaps pluralise the Namespace to Vehicles, and Customers in order to remove this duplication (or drop the namespace).